src/shared/mount-util.c

```diff
@@ -1386,7 +1386,8 @@ int make_userns(uid_t uid_shift, uid_t uid_range, uid_t source_owner, uid_t dest
 
 int remount_idmap_fd(
                 char **paths,
-                int userns_fd) {
+                int userns_fd,
+                int extra_set_flags) {
```

Comment:
`mount_attr.attr_set` is `__u64`. So, please use `uint64_t` here.
Also, maybe, better to name `extra_mount_attr_set` ?